pipeline.yml

```diff
@@ -16,6 +16,8 @@ jobs:
     - get: bosh-stemcell
     - get: common-staging
       trigger: true
+    - get: common-releases-tooling-verion
```

Comment:
might want to do some double checking here, should be `-version`. Several others like this.